cmd/csi-driver/Dockerfile

```diff
@@ -1,7 +1,12 @@
 FROM registry.access.redhat.com/ubi7/ubi-minimal:7.9-256
 
+RUN microdnf install -y yum
 RUN microdnf update -y && rm -rf /var/cache/yum
+RUN curl https://rpmfind.net/linux/centos/7.9.2009/os/x86_64/Packages/cryptsetup-2.0.3-6.el7.x86_64.rpm -o cryptsetup-2.0.3-6.el7.x86_64.rpm
+RUN yum install -y cryptsetup-2.0.3-6.el7.x86_64.rpm
+RUN rm -f cryptsetup-2.0.3-6.el7.x86_64.rpm
 RUN microdnf install -y systemd
+RUN microdnf remove -y yum
```

Comment:
This is very ugly. Please add a file named `CentOS-Base.repo` to this repo with the following content:
```
[base]
name=CentOS-7 - Base
mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=os&infra=$infra
gpgcheck=0
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7

[updates]
name=CentOS-7 - Updates
mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=updates&infra=$infra
gpgcheck=0
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
```

Then replace line 3-9 with:
```
ADD CentOS-Base.repo /etc/yum.repos.d/
```
Line 4 (currently 8) should then read:
```
RUN microdnf install -y systemd cryptsetup
```
This is the pattern we've established with other external dependencies (such as NFS).

Reply:
@datamattsson , we are not adding any CentOS repository package for NFS provisioner , rather we pull the nfs provisioner image as a container and we run ganeshd as a container in pod

https://github.com/hpe-storage/csi-driver/blob/be80c2e6252d89f8d463579ab1338390e7ad7296/pkg/flavor/kubernetes/nfs.go#L27

We will explore the option of adding this centos repository as suggested by you.

Reply:
I was merely referring to the pattern of adding packages from outside of what is available in the UBI repo.

Reply:
Thanks @datamattsson - nice input! Incorporated it locally and did some testing. Once all the review comments are addressed, I will push the changes.